### PR TITLE
New dialogs

### PR DESCRIPTION
**Pull request by Lunalo**
This is now working with current code.

### CODE REVIEW

File: instat/dlgRowStats.vb [hunk 1]
```diff
@@ -40,21 +71,19 @@ Public Class dlgRowStats
...
+            ElseIf rdoStandardDeviation.Checked = True Then
+                ucrBase.clsRsyntax.AddParameter("FUN", "sd")
+            Else
+                ucrBase.clsRsyntax.RemoveParameter("FUN")
         End If
 
     End Sub
 
-    Private Sub ucrNewColumnNameSelector_Leave(sender As Object, e As EventArgs) Handles ucrNewColumnNameSelector.Leave
-        ucrBase.clsRsyntax.SetAssignTo(strAssignToName:=ucrNewColumnNameSelector.cboColumnName.Text, strTempDataframe:=ucrSelectorByDataFrameAddRemove.ucrAvailableDataFrames.cboAvailableDataFrames.Text, strTempColumn:=ucrNewColumnNameSelector.cboColumnName.Text)
```
Comment (dannyparsons) [inline instat/dlgRowStats.vb]: Why did you get rid of this? We still need it to update the assignTo when the user types something different

Comment (Lunalo) [inline instat/dlgRowStats.vb]: Hi Danny

There has been a power cut here but now the power is back.
I understand your point here. The changes will be made in short while.

Thanks.

Regards,
John Ong'ala Lunalo|| Kenyatta University Alumnus||
Volunteer, African Maths Initiative||
Email Address :johnlunalo95@gmail.com
Tel: +254701519572, +254786438572

Comment (dannyparsons) [inline instat/dlgRowStats.vb]: OK thanks, I guessed that might have happened since everyone went quiet!

Comment (Lunalo) [inline instat/dlgRowStats.vb]: I have changed that already, and a question for you in your mail

Regards,
John Ong'ala Lunalo|| Kenyatta University Alumnus||
Volunteer, African Maths Initiative||
Email Address :johnlunalo95@gmail.com
Tel: +254701519572, +254786438572

**Pull request merged**